app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionDetailFragment.java

```diff
@@ -64,14 +65,35 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         AppUtil.setTaskDescription(activity, session.title, ContextCompat.getColor(activity, session.category.getVividColorResId()));
     }
 
+    // Receive all viewModel's properties changed.
+    private final Observable.OnPropertyChangedCallback onPropertyChanged = new Observable.OnPropertyChangedCallback() {
```

Comment:
Thanks for your explanation! Got it 😄 
But in Android, I guess it is better to use EventBus. Because Android has specific lifecycle and it is difficult to add `OnPropertyChanged` callback to previous Activity.
I'm so sorry 🙇